sdk/src/main/java/com/mapbox/maps/viewannotation/ViewAnnotationManager.kt

```diff
@@ -155,6 +155,17 @@ interface ViewAnnotationManager {
    */
   fun getViewAnnotationUpdateMode(): ViewAnnotationUpdateMode
 
+  /**
+   * Retrieve added [View] with their [ViewAnnotationOptions]. This will return
+   * fresh copies of [View] and associated [ViewAnnotationOptions] every time.
+   *
+   * Note: Modifying [ViewAnnotationOptions] will not update actual annotation options. Use
+   * [ViewAnnotationManager.updateViewAnnotation] instead. Modifying [View] will update the actual view property.
+   *
+   * @return Map of [View] and associated [ViewAnnotationOptions].
+   */
+  val annotations: HashMap<View, ViewAnnotationOptions>
```

Comment:
Any reasons to return specific HashMap and not just plain Map?

Reply:
Yeah, Interface sounds good. 👍